src/Resource/NullResource.php

```diff
@@ -0,0 +1,14 @@
+<?php namespace League\Fractal\Resource;
+
+class NullResource extends ResourceAbstract
+{
+    /**
+     * Get the data.
+     *
+     * @return mixed
+     */
+    public function getData()
+    {
+        return null;
```

Comment:
I am more on the side of @burriko on that specific piece of code.

But it seems to be the a rule that is followed in this project, since I couldn't find any other instances of `return null` in the codebase.

@jasonlewis Since you are the project maintainer, it is your call: Should I remove the statement or not?